src/model.jl

```diff
@@ -19,7 +19,11 @@ function Model(mesh, num_steps_year; co2_concentration = 315.0) # co2_concentrat
     # Read parameters
     
     geography = read_geography(joinpath(@__DIR__, "..", "input", "The_World.dat"),nx,ny)
-    albedo    = read_albedo(joinpath(@__DIR__, "..", "input", "albedo.dat"),nx,ny)
+    # Read data from input
+    #input_albedo = read_albedo(joinpath(@__DIR__, "..", "input", "albedo.dat"),nx,ny)
```

Comment:
It would be good that the user can specify if they provide a file with the albedo, or if they want the albedo to be computed from the geography.

Reply:
@xibach Please resolve once the change is implemented :)